bin/configs/go-server-no-body-path-params.yaml

```diff
@@ -0,0 +1,8 @@
+generatorName: go-server
+outputDir: samples/server/others/go-server/no-body-path-params
```

Comment:
please add the output folder to ` .github/workflows/samples-go.yaml` so that the CI can test the output moving forward?

Reply:
Done